react/week5/Github_Users_Search/src/index.js

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter, Route, Switch, Link } from 'react-router-dom';
```

Comment:
Hi Swathi,
Please review the documentation for the import BrowserRouter.
https://reacttraining.com/react-router/web/api/BrowserRouter

Reply:
Hi Ricardo,
I have changed the usage as in the document. But just to know is there a specific reason why we do that?